### PR TITLE
fix(PhoneInput): render flags correctly when dezoomed

### PR DESCRIPTION
**Pull request by QuiiBz**
## Type

- Bug

#### The following changes where made:

Fix PhoneInput component that rendered flags incorrectly when page in dezoomed.
Check if it works and if it still works when zooming.

## Relevant logs and/or screenshots

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| ?path=/docs/components-data-entry-phoneinput--default  | Screenshot 2022-07-19 at 12 24 10 | Screenshot 2022-07-19 at 12 23 19 |

### CODE REVIEW

File: src/components/PhoneInput/index.tsx [hunk 1]
```diff
@@ -80,10 +79,6 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
     .iti__selected-dial-code {
       padding-left: 5px;
     }
-
-    .iti__flag {
-      background-image: url(${flags});
```
Comment (chambo-e) [inline src/components/PhoneInput/index.tsx:85]: The issue is that it won't work on project that use scw-ui as flags provided by intl-tel-input are relative paths, we need to explicitly tell it where to find the flags

https://github.com/jackocnr/intl-tel-input#getting-started-using-a-bundler-eg-webpack

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: src/components/PhoneInput/index.tsx (hunk 1 above)
Comment (QuiiBz) [reply src/components/PhoneInput/index.tsx]: Fixed by using the `flags.png` image instead of `flags@2x` that was used as `flags.png`

Comment (chambo-e) [reply src/components/PhoneInput/index.tsx]: So we loose support for high dpi devices ? 😄

Comment (QuiiBz) [reply src/components/PhoneInput/index.tsx:85]: Pushed a commit that adds back the `flags@2x` image, so we're back to the state before.

**Comment by QuiiBz** [comment]
Closing since we will remove this component soon from `scaleway-ui`

**Pull request closed without merging**